cmd/brew-test-bot.rb

```diff
@@ -812,11 +812,7 @@ def deleted_formula(formula_name)
     def coverage_args
       return [] unless ARGV.include?("--coverage")
       return [] if @test_bot_tap
```

Comment:
Let's remove this line. It's my fault rather than yours but this is why we didn't catch this before now and couldn't reproduce on this tap (which worried me).